dan_layer/core/src/models/instruction.rs

```diff
@@ -78,6 +81,10 @@ impl Instruction {
         &self.args
     }
 
+    pub fn sender(&self) -> PublicKey {
```

Comment:
nit: this naming convention usually means a reference to the item is returned.
Suggest either `to_sender_public_key` or return a reference.

Reply:
Ah yeah, I thought it was `Copy` for some reason....